src/IoC/ServiceExtensions.cs

```diff
@@ -161,7 +163,28 @@ public static IServiceCollection AddServices(this IServiceCollection services)
                 .AddTransient<IPdfService>(x => new PdfService(ConfigurationManager.Configuration["PDF_SERVICE_ROOT"], new HttpClient()))
                 .AddTransient<IReportingHelper, ReportingHelper>()
                 .AddTransient<IPurchaseOrdersReportService, PurchaseOrdersReportService>()
-                .AddTransient<IPurchaseOrderService, PurchaseOrderService>()
+                .AddTransient<IPurchaseOrderService>(
+                    x => new PurchaseOrderService(
+                    ConfigurationManager.Configuration["APP_ROOT"],
```

Comment:
this is a bit crazy huh? I wonder if there's a neater way

Reply:
yeah bit painful when all I need that it can't get on its own is app root